src/nvim/api/vim.c

```diff
@@ -2450,3 +2451,191 @@ void nvim_del_user_command(String name, Error *err)
 {
   nvim_buf_del_user_command(-1, name, err);
 }
+
+/// Parse command line.
+///
+/// Doesn't check the validity of command arguments.
+///
+/// @param str       Command line string to parse. Cannot contain "\n".
+/// @param opts      Optional parameters. Reserved for future use.
+/// @param[out] err  Error details, if any.
+/// @return Dictionary containing command information, with these keys:
+///         - cmd: (string) Command name.
+///         - line1: (number) Starting line of command range.
+///         - line2: (number) Final line of command range.
+///         - force: (boolean) Whether command contains a bang.
+///         - args: (array) Command arguments.
+///         - addr: (string) Value of |:command-addr|. Uses short name.
+///         - nargs: (string) Value of |:command-nargs|.
+///         - nextcmd: (string) Next command if there are multiple commands separated by a |:bar|.
+///                             Empty if there isn't a next command.
+///         - magic: (array) Which characters have special meaning inside the command arguments.
+///                          Possible values inside the array are:
+///             - '%%': '%%' and '#' get expanded by the command.
+///             - '|': '|' may get treated specially by the command depending on the command and
```

Comment:
single-quote in a vimdoc is reserved for options. So use double quotes.
```suggestion
///             - "|": "|" may get treated specially by the command depending on the command and
```

Reply:
What should we do about the `"` character then, we can't put it inside double quotes

Reply:
i thought we didn't need `"` as a magic char? If it is needed I'd write `double-quote (")`

Reply:
> i thought we didn't need `"` as a magic char? If it is needed I'd write `double-quote (")`

We didn't, but we still need to reference that `"|"` is related to it in the docs

Reply:
Anyway it's fixed now